### DISCUSSION TITLE
Configure my jenkins pipeline to stop/fails when dependency-track detects vulnerabilities

### DISCUSSION THREAD

**Discussion by MehdiAmdouni** [DependencyTrack/dependency-track#2096] [Q&A]
Hi there, I have a "quick" question for you guys.
So i have dependency-track running as a container and i generated/published BOM file using jenkins plugin and after that the pipeline goes to the deployment stage, and it's all working fine.
My question is there any possible way that i can configure my jenkins pipeline to stop/fails when dependency-track detects vulnerabilities? instead of just analyzing and passing to the deployment stage anyway? i don't want to go to the deployment stage when i have some vulnerabilities detected.

Thanks in advance.

**Comment by syalioune** [comment]
Hello @MehdiAmdouni,
It's best to use [Discussions](https://github.com/DependencyTrack/dependency-track/discussions) for questions.

Nevertheless, I believe Dependency Track Jenkins plugin already provide exactly that capability through **synchronous upload** and [Thresholds](https://plugins.jenkins.io/dependency-track/#plugin-content-thresholds).
